### PR TITLE
A couple more Coverity-related tweaks

### LINKED ISSUE #30: dfuzzer seems to fail to compile with -Werror
```
[1/6] clang -Idfuzzer.p -I. -I.. -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/libmount -I/usr/include/blkid -fcolor-diagnostics -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=gnu11 -O0 -g -Werror -pthread -MD -MQ dfuzzer.p/src_introspection.c.o -MF dfuzzer.p/src_introspection.c.o.d -o dfuzzer.p/src_introspection.c.o -c ../src/introspection.c
FAILED: dfuzzer.p/src_introspection.c.o
clang -Idfuzzer.p -I. -I.. -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/libmount -I/usr/include/blkid -fcolor-diagnostics -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=gnu11 -O0 -g -Werror -pthread -MD -MQ dfuzzer.p/src_introspection.c.o -MF dfuzzer.p/src_introspection.c.o.d -o dfuzzer.p/src_introspection.c.o -c ../src/introspection.c
../src/introspection.c:50:59: error: unused parameter 'name' [-Werror,-Wunused-parameter]
int df_init_introspection(GDBusProxy *dproxy, const char *name, const char *interface)
                                                          ^
1 error generated.
```

### PR DESCRIPTION
**Pull request by mrc0mmand**
The code annotations for `rand()` and `random()` feel a bit icky (I wish there was an option to disable the warning for the whole file), but it is what it is. I guess the other option would be to suppress them manually in the UI.

The remaining Coverity warnings will require a bit more work.

Also, this fixes #30 - the `name` parameter was originally used just in the error paths, but the `name` argument was dropped in 15a6d7b9f60269fdf5e3febb61eb45e481873a8c, so was completely unused since then.

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -12,9 +12,6 @@ libffi = dependency('libffi', required : true)
 
 subdir('src')
 
-# https://github.com/matusmarhefka/dfuzzer/issues/30
-add_project_arguments('-Wno-unused-parameter', language : 'c')
```
Comment (evverx) [inline meson.build:16]: Could you remove it from `Makefile` to fully unleash `-Werror` on Packit as well?

**Comment by evverx** [comment]
> The code annotations for rand() and random() feel a bit icky (I wish there was an option to disable the warning for the whole file), but it is what it is

I'm not sure those annotations should pollute the code. I went through all the "risky function" warnings and marked them as "intentional" so they are no longer shown by default.

**Comment by mrc0mmand** [comment]
> > The code annotations for rand() and random() feel a bit icky (I wish there was an option to disable the warning for the whole file), but it is what it is
> 
> I'm not sure those annotations should pollute the code. I went through all the "risky function" warnings and marked them as "intentional" so they are no longer shown by default.

Yeah, I have the same feeling, thanks. I'll drop the commit.

**Comment by evverx** [comment]
FWIW Looks like Coverity still complains about the comma.

**Pull request merged**